src/main/java/mate/academy/internetshop/dao/UserDao.java

```diff
@@ -1,7 +1,13 @@
 package mate.academy.internetshop.dao;
 
+import java.util.Optional;
+import mate.academy.internetshop.exeptions.AuthenticationException;
 import mate.academy.internetshop.model.User;
 
 public interface UserDao extends GenericDao<User, Long> {
 
+    Optional<User> findByLogin(String login, String password) throws AuthenticationException;
```

Comment:
на рівні даю використовуємо інший метод (писала в чат про нього)
```suggestion
    Optional<User> findByLogin(String login);
```